internal/cmd/stack/environment.go

```diff
@@ -73,6 +119,89 @@ func setVar(cliCtx *cli.Context) error {
 	return nil
 }
 
+func (e *listEnvCommand) listEnv(cliCtx *cli.Context) error {
+	outputFormat, err := cmd.GetOutputFormat(cliCtx)
+	if err != nil {
+		return err
+	}
+
+	stackID := cliCtx.String(flagStackID.Name)
+
+	var query listEnvQuery
+	variables := map[string]interface{}{
+		"stack": graphql.ID(stackID),
+	}
+
+	if err := authenticated.Client.Query(cliCtx.Context, &query, variables); err != nil {
+		return err
+	}
+
+	var outputs []listEnvElementOutput
```

Comment:
nit: can you rename this to `elements` or something like that?

Reply:
Sure